Lib/random.py

```diff
@@ -739,6 +739,12 @@ def getrandbits(self, k):
         x = int.from_bytes(_urandom(numbytes), 'big')
         return x >> (numbytes * 8 - k)                # trim excess bits
 
+    def randbytes(self, n):
+        """Generate n random bytes."""
+        if n < 0:
+            raise ValueError('number of bytes must be non-negative')
```

Comment:
It is not necessary. `os.urandom()` already raises an exception with correct type and message.